solr/core/src/test/org/apache/solr/update/SolrIndexConfigTest.java

```diff
@@ -118,7 +118,7 @@ public void testTieredMPSolrIndexConfigCreation() throws Exception {
     ConcurrentMergeScheduler ms = (ConcurrentMergeScheduler) iwc.getMergeScheduler();
     assertEquals("ms.maxMergeCount", 987, ms.getMaxMergeCount());
     assertEquals("ms.maxThreadCount", 42, ms.getMaxThreadCount());
-    assertTrue("ms.isAutoIOThrottle", ms.getAutoIOThrottle());
+    assertFalse("ms.isAutoIOThrottle", ms.getAutoIOThrottle());
```

Comment:
note-to-self bookmark: would like to understand more about this change here.

Reply:
Looks like Lucene's default changed and so something like #3459 is needed to match.